test/e2e/egress_test.go

```diff
@@ -664,6 +666,89 @@ func testEgressMigration(t *testing.T, data *TestData, triggerFunc, revertFunc f
 	checkIPNeighbor(fromNode)
 }
 
+func testEgressUpdateBandwidth(t *testing.T, data *TestData) {
```

Comment:
I am a bit cautious about that kind of test, as flakes seem likely. Do you think it will succeed consistently?

Reply:
I tested in my local env several times and all passed. But for the burst test case, it only runs 1s, including the connection's initialization. There is a chance to be flaky. Do you think we could go with a larger epsilon or something like do several times with one grace fail?

Reply:
Your local environment is likely less flaky than the Github Actions runners.

I can also see the test failing if the OVS meter is not updated fast enough when you update the Egress bandwidth with the K8s API. There is no delay in the test there (unlike after `CREATE`, where we can use `waitForEgressRealized`).

I feel like for e2e testing, we may want to test a single bandwidth, and remove the `UPDATE`. Hopefully we have good coverage in unit tests for bandwidth update and bandwidth removal? After that we will see in time if the test is flaky or not in CI.

Reply:
BTW, if the burst case is already flaky, just run iperf for longer (5 or 10s) and use a very large burst value.

For example, use `rate=100Mbps` and `burst=1G`, and run the iperf command for 10s. I would also use a 20% epsilon for both cases, to be on the safe side.

Reply:
We do have UT cases that cover bandwidth updates and removal. Changed to single bandwidth test in e2e.

I haven't seen the burst case flaky. Running it for longer is a good idea for me. But I suggest we keep it before we see it flaky, after all running for a longer time is not that "accurate" for burst measurement. What do you think?

Reply:
I think by running it for longer, you ensure that the "committed rate" part of the throughput is closer to the target value (percentage wise), so the increased accuracy can come from there. But I'll let you decide.

Reply:
Understood. Then let's see if flaky does exist first.